packages/tasks/src/model-libraries.ts

```diff
@@ -90,6 +90,7 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/facebookresearch/audiocraft",
 		snippets: snippets.audiocraft,
 		filter: false,
+		countDownloads: `path:"state_dict.bin" OR path:"config.json"`,
```

Comment:
At the moment, the current download setup will just take into account the primary library for download stats. That means that if a repo is both Audiocraft and transformers, and transformers is the primary library, audiocraft checkpoints will not be taken into account. (there's an issue for this)

Due to this, I would make this filter just about audiocraft (so just keep `path:"state_dict.bin"`). WDYT?

Reply:
Also in favor of just keeping `state_dict.bin`